docs/book/src/topics/managedcluster.md

```diff
@@ -214,3 +239,83 @@ Current limitations
 - Only supports Standard load balancer (SLB).
   - We will not support Basic load balancer in CAPZ. SLB is generally
     the path forward in Azure.
+
+## Troubleshooting
+
+1.) If a user tries to delete the machinepool which refers to the last system node pool AMMP webhook will reject deletion, so time stamp never gets set on the AMMP. However the timestamp would be set on the MP and would be in deletion state. To recover from this state create a new MP manually referencing the AMMP, fudge required references and finalizers in MP. In the AMMP remove the owner reference to the old MP, and set it to the new MP. Once the new MP is pointing to the AMMP you can delete the old MP. To delete the old MP remove the finalizers in that object.
```

Comment:
nit: as much as I like "fudge required references and finalizers", _slightly_ more professional phrasing might be nice 🙂 

perhaps: "manually edit the required reference and finalizers to link a new MachinePool to the AzureManagedMachinePool and unlink the old MachinePool"

Reply:
ack !

Reply:
great docs, by the way!